blocklib/picoscope4000a/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+set(PICOSCOPE4000A_SOURCES
+    picoscope4000a.cc
+)
+
+set(PICOSCOPE4000A_HEADERS
+    picoscope4000a.hpp
+)
+
+# TODO do not hardcode
+add_library(ps4000a SHARED IMPORTED)
+set_property(TARGET ps4000a PROPERTY
+             IMPORTED_LOCATION ${PICOSCOPE_PREFIX}/lib/libps4000a.so)
+target_include_directories(ps4000a INTERFACE ${PICOSCOPE_PREFIX}/include/libps4000a ${PICOSCOPE_PREFIX}/include/libps5000a) # Hack: PicoCallback.h is missing in libps4000a/
+
+add_library(gr-picoscope4000a SHARED ${PICOSCOPE4000A_SOURCES})
```

Comment:
The problem is that the ps4000Api.h itself has includes without prefix:

    #include "PicoStatus.h"
    #include "PicoCallback.h"
    #include "PicoConnectProbes.h"
    #include "PicoVersion.h"

So I wouldn't know how one could include more than one header-only impl without these clashing (the build pulling in whatever is listed first via `-I`). Right now these headers are identical in subfolders (and some are even missing in libps4000a...), but that's just coincidental, I think.

Reply:
due to the quote includes, these will be resolved in the directory of the including file, so the -I path should only be used in case the file is not found there. The Include guards clash though, so if 2 models have different versions of the includes, which one is actually used depends on inclusion order...